content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Add custom methods to your models'
+metaTitle: 'Prisma Client extensions: model (Preview)'
+metaDescription: 'Extend the functionality of Prisma Client'
+tocDepth: 3
+---
+
+<TopBlock>
+
+<Admonition type="warning">
+
+This page is in draft.
+
+</Admonition>
+
+From version **TBD**, you can use `model` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) to add custom methods to your models.
+
+Use the `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods) to create an _extended client_. An extended client is a forked variant of the standard Prisma Client with an extension applied. Use the `model` extension type to add methods to a specific model in your schema or to all models in your schema.
```

Comment:
I do want to repeat it because a user might drop straight into this page if they search for adding custom methods to models (the same applies to the other "extension type" pages). We've got a link to the general extensions page if the person needs that prerequisite information and a short description of what extensions are.

My goal is for this to be enough information for them to set this a method on a model from this page alone, but if they want more information on how extensions work, they can go to the first page.